src/irr/asset/CGLILoader.cpp

```diff
@@ -0,0 +1,552 @@
+				case gl::INTERNAL_RGBA_ETC2: return getTranslatedFinalFormat(EF_ETC2_R8G8B8A8_UNORM_BLOCK);					//GL_COMPRESSED_RGBA8_ETC2_EAC
+				case gl::INTERNAL_R11_EAC: return getTranslatedFinalFormat(EF_EAC_R11_UNORM_BLOCK);						//GL_COMPRESSED_R11_EAC
+				case gl::INTERNAL_SIGNED_R11_EAC: return getTranslatedFinalFormat(EF_EAC_R11_SNORM_BLOCK);				//GL_COMPRESSED_SIGNED_R11_EAC
+				case gl::INTERNAL_RG11_EAC: return getTranslatedFinalFormat(EF_EAC_R11G11_UNORM_BLOCK);						//GL_COMPRESSED_RG11_EAC
+				case gl::INTERNAL_SIGNED_RG11_EAC: return getTranslatedFinalFormat(EF_EAC_R11G11_SNORM_BLOCK);				//GL_COMPRESSED_SIGNED_RG11_EAC
+
+				case gl::INTERNAL_RGBA_ASTC_4x4: return getTranslatedFinalFormat(EF_ASTC_4x4_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_4x4_KHR
+				case gl::INTERNAL_RGBA_ASTC_5x4: return getTranslatedFinalFormat(EF_ASTC_5x4_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_5x4_KHR
+				case gl::INTERNAL_RGBA_ASTC_5x5: return getTranslatedFinalFormat(EF_ASTC_5x5_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_5x5_KHR
+				case gl::INTERNAL_RGBA_ASTC_6x5: return getTranslatedFinalFormat(EF_ASTC_6x5_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_6x5_KHR
+				case gl::INTERNAL_RGBA_ASTC_6x6: return getTranslatedFinalFormat(EF_ASTC_6x6_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_6x6_KHR
+				case gl::INTERNAL_RGBA_ASTC_8x5: return getTranslatedFinalFormat(EF_ASTC_8x5_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_8x5_KHR
+				case gl::INTERNAL_RGBA_ASTC_8x6: return getTranslatedFinalFormat(EF_ASTC_8x6_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_8x6_KHR
+				case gl::INTERNAL_RGBA_ASTC_8x8: return getTranslatedFinalFormat(EF_ASTC_8x8_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_8x8_KHR
+				case gl::INTERNAL_RGBA_ASTC_10x5: return getTranslatedFinalFormat(EF_ASTC_10x5_UNORM_BLOCK); 				//GL_COMPRESSED_RGBA_ASTC_10x5_KHR
+				case gl::INTERNAL_RGBA_ASTC_10x6: return getTranslatedFinalFormat(EF_ASTC_10x6_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_10x6_KHR
+				case gl::INTERNAL_RGBA_ASTC_10x8: return getTranslatedFinalFormat(EF_ASTC_10x8_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_10x8_KHR
+				case gl::INTERNAL_RGBA_ASTC_10x10: return getTranslatedFinalFormat(EF_ASTC_10x10_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_10x10_KHR
+				case gl::INTERNAL_RGBA_ASTC_12x10: return getTranslatedFinalFormat(EF_ASTC_12x10_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_12x10_KHR
+				case gl::INTERNAL_RGBA_ASTC_12x12: return getTranslatedFinalFormat(EF_ASTC_12x12_UNORM_BLOCK);				//GL_COMPRESSED_RGBA_ASTC_12x12_KHR
+
+				// sRGB formats
+				case gl::INTERNAL_SRGB_DXT1: return getTranslatedFinalFormat();					//GL_COMPRESSED_SRGB_S3TC_DXT1_EXT
+				case gl::INTERNAL_SRGB_ALPHA_DXT1: return getTranslatedFinalFormat();				//GL_COMPRESSED_SRGB_ALPHA_S3TC_DXT1_EXT
+				case gl::INTERNAL_SRGB_ALPHA_DXT3: return getTranslatedFinalFormat();				//GL_COMPRESSED_SRGB_ALPHA_S3TC_DXT3_EXT
```

Comment:
https://github.com/buildaworldnet/IrrlichtBAW/blob/master/include/irr/asset/format/EFormat.h#L157

Reply:
DXT3 and BC2? Is it valid?

Reply:
DXT3 is BC2, google it